relayer/workers/parachaincommitmentrelayer/main.go

```diff
@@ -11,20 +11,18 @@ import (
 	"github.com/snowfork/polkadot-ethereum/relayer/chain/parachain"
 	"github.com/snowfork/polkadot-ethereum/relayer/chain/relaychain"
 	"github.com/snowfork/polkadot-ethereum/relayer/crypto/secp256k1"
-	"github.com/snowfork/polkadot-ethereum/relayer/workers/parachaincommitmentrelayer/parachaincommitment"
 )
 
 type Worker struct {
-	parachainConfig             *parachain.Config
-	relaychainConfig            *relaychain.Config
-	ethereumConfig              *ethereum.Config
-	parachainConn               *parachain.Connection
-	relaychainConn              *relaychain.Connection
-	parachainCommitmentListener *parachaincommitment.Listener
-	ethereumConn                *ethereum.Connection
-	ethereumChannelWriter       *EthereumChannelWriter
-	beefyRelaychainListener     *BeefyListener
-	log                         *logrus.Entry
+	parachainConfig       *parachain.Config
```

Comment:
Nit: A lot of accumulation of state here into a "God" object. In a follow-up PR, may be worth refactoring in such a way that the commitment relayer delegates to child listeners and writers, communicating via channels. 

For example, the `relaychainConfig` field can probably be removed and pushed down into the beefy listener.